monkey/monkey_island/cc/ui/src/components/attack/techniques/MitigationsComponent.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactTable from 'react-table';
+import marked from 'marked';
+import '../../../styles/report/AttackReport.scss';
+
+
+class MitigationsComponent extends React.Component {
+
+  constructor(props) {
+    super(props);
+    if (typeof this.props.mitigations !== 'undefined'){
+      let descriptions = MitigationsComponent.parseDescription(this.props.mitigations.description);
+      this.state = {name: this.props.mitigations.name, descriptions: descriptions};
+    } else {
+      this.state = {name: '', descriptions: []}
+    }
+  }
+
+  static parseDescription(description){
+    const citationRegex = /\(Citation:.*\)/gi;
+    const emptyLineRegex = /^\s*[\r\n]/gm;
+    description = description.replace(citationRegex, '');
+    description = description.replace(emptyLineRegex, '');
+    let descriptions = description.split('\n');
+    descriptions = descriptions.map(function(paragraph){ return marked(paragraph); });
+    return descriptions;
+  }
+
+  static getMitigationDescriptions(name) {
+    return ([{
+      Header: name,
+      style: {'text-align': 'left'},
+      columns: [
+        { id: 'description',
+          accessor: x => (<div dangerouslySetInnerHTML={{__html: x}} />),
+          style: {'whiteSpace': 'unset'}}
+      ]
+    }])
+  }
```

Comment:
Missing the mitigation name + link before the description 
![image](https://user-images.githubusercontent.com/48879847/77849525-e360d180-71d4-11ea-90bd-958721373375.png)